__tests__/app.test.js

```diff
@@ -87,3 +76,39 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of all the articles which contains the correct article data", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+            article_img_url: expect.any(String),
+          });
+          expect(Number(article.comment_count)).not.toBeNaN();
+        });
+      });
+  });
+});
```

Comment:
Nice, great test. I like the length test too. I thing the NaN test isn't really necessary, when it comes to front end, this will comes through as a string anyway. I like the outside the box thinking though.